pow-migration/src/main.rs

```diff
@@ -21,8 +23,15 @@ struct Args {
     /// Path to the PoS configuration file
     #[arg(short, long)]
     config: String,
+    /// Output a list of registered validators
     #[arg(short, long)]
     list_validators: bool,
+    /// Output a list of pre-stakers
+    #[arg(short = 's', long)]
+    list_stakers: bool,
+    /// Only show the pre-stakers for a specific validator. This argument is only intended to be used in conjunction with list-stakers
```

Comment:
I believe we should make this explicit in the `clap` parser by using an nested struct. For instance the help argument wouldn't be able to document this.

Reply:
Fixed